lib/web/app/handler.go

```diff
@@ -157,9 +161,6 @@ func (h *Handler) handleFragment(w http.ResponseWriter, r *http.Request) error {
 			SameSite: http.SameSiteLaxMode,
 		})
 
-		// Set additional security headers. In the first run, only set strict
-		// transport security. In the future we can add other headers that make sense.
-		w.Header().Add("Strict-Transport-Security", "max-age=31536000; includeSubDomains")
```

Comment:
why is this removed?

Reply:
For the POST endpoint this header is not relevant. It is for web pages to enforce HTTPS on loads and for protect against redirects which we do set on redirect page itself.